app_backend/tests/favorites_test.py

```diff
@@ -7,25 +7,23 @@
 
 def test_toggle_favorite():
     class Params:
-        def __init__(self, food_id: str, state: int, expected_status_code: int, expects_err: bool):
-            self.food_id = food_id
+        def __init__(self, food_name: str, state: bool, expected_status_code: int, expects_err: bool):
+            self.food_name = food_name
```

Comment:
why are we doing food name instead of food id?

Reply:
There's no natural food_id. We can technically hash the food name, but we might as well just use the food name as a natural form of identification